x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -209,7 +214,12 @@ export const uiQueryParams: (
 
         if (value !== undefined) {
           if (key === 'show') {
-            if (value === 'policy_response' || value === 'details' || value === 'isolate') {
```

Comment:
getting a bit long. Can probably refactor ( `['...',].includes(value)` is probably fine, or extract up if it makes sense)

Reply:
> This doesn't seem to include action-responses yet, is that going to be a different PR?

Yes, I'll do that in the PR for the API that we discussed. Else the types were getting complex and hence more changes in this PR.